src/main/resources/assets/patchouli/lang/de_DE.lang

```diff
@@ -0,0 +1,93 @@
+
+
+#Subtitles
+patchouli.subtitle.book_open=Buch, das geöffnet wird
+patchouli.subtitle.book_flip=Seitenumschläge
+
+
+
+# Gui Strings
+patchouli.gui.lexicon.landing_info=(Platzhalter für den Einführungstext, bestimme dein eigenes in der json-Datei deines Buches!)
+patchouli.gui.lexicon.progress_meter=Fortschritt freischalten:
+patchouli.gui.lexicon.progress_tooltip=%d/%d Freigeschaltet
+patchouli.gui.lexicon.progress_tooltip.secret1=Plus 1 Geheimnis
+patchouli.gui.lexicon.progress_tooltip.secret=Plus %d Geheimnisse
+patchouli.gui.lexicon.progress_tooltip.info=Mache Fortschritte um noch mehr freizuschalten!
+patchouli.gui.lexicon.categories=Kategorien
+patchouli.gui.lexicon.chapters=Kapitel
+patchouli.gui.lexicon.editor=Buch-Bearbeitungsprogramm
+patchouli.gui.lexicon.editor.mock_header=Kopfzeile der Nachbildung
+patchouli.gui.lexicon.editor.info=Dies ist ein Bearbeitungsprogramm für Patchouli's-Bucheinträge. Damit ist gemeint, dass es für die Entwicklung oder Übersetzung genutzt wird. Es gibt dafür kein Nutzen wenn du ein Spieler bist.$(br2)Bitte achte auf dem $(l:https://github.com/Vazkii/Patchouli/wiki/Text-Formatting-101)Patchouli Wiki$() für nützliche Steuercodes.
+patchouli.gui.lexicon.shift_for_recipe=Umschalttaste-klicken für Rezept
+patchouli.gui.lexicon.index=Eintragsverzeichnis
+patchouli.gui.lexicon.index.info=Ein Verzeichnis von jedem Kapitel ist Verfügbar in diesem Buch.$(br2)Du kannst den Verzeichnis durchsuchen oder irgendwelche andere Kategorie, ganz einfach durch das Eintippen der Anfrage. Die Suchleiste wird dann erscheinen um dich zu unterstützen.
+patchouli.gui.lexicon.shapeless=Formlos
+patchouli.gui.lexicon.no_results=Keine Ergebnisse
+patchouli.gui.lexicon.sad=:(
+patchouli.gui.lexicon.locked=(Gesperrt)
+patchouli.gui.lexicon.reloaded=Buch neugeladen. Dauerte %d ms.
+patchouli.gui.lexicon.add_bookmark=Lesezeichen hinzufügen
+patchouli.gui.lexicon.remove_bookmark=(Umschalttaste drücken und klicken zum entfernen)
+patchouli.gui.lexicon.multiblock_bookmark=Visualisierte Struktur
+patchouli.gui.lexicon.relations=Verwandte Kapitel
+patchouli.gui.lexicon.not_anchored=Rechts-klicken auf einen Block um die Struktur zu fixieren
+patchouli.gui.lexicon.structure_complete=Fertig!
+patchouli.gui.lexicon.needs_air=(Lösche Blöcke die rot markiert sind)
+patchouli.gui.lexicon.visualize_letter=V
+patchouli.gui.lexicon.seconds=%ss
+patchouli.gui.lexicon.history=Lese-Verlauf
+patchouli.gui.lexicon.history.info=Dieser Bereich speichert einige der vorherigen Kapiteln die du besucht hast.$(br2)Kapiteln werden automatisch hinzugefügt und entfernt während du das Buch durchblätterst. Willst du aber, dass sie länger referenziert bleiben, kannst du sie als Lesezeichen hinzufügen. $(br2)$(o)Nützlicher Hinweis: Versuche mal die Umschalttaste zu drücken und klicke auf der Kapitel-Schaltfläche!$()
```

Comment:
The plural of "Kapitel" is "Kapitel", not "Kapiteln"... And you're missing punctuation again.
```suggestion
patchouli.gui.lexicon.history.info=Dieser Bereich speichert die vorherigen paar Kapitel, die du besucht hast.$(br2)Kapitel werden automatisch hinzugefügt und entfernt, während du das Buch durchblätterst. Willst du aber, dass sie länger gelistet bleiben, kannst du sie als Lesezeichen hinzufügen. $(br2)$(o)Nützlicher Hinweis: Versuche mal, die Umschalttaste zu drücken, und dann dann auf die Kapitel-Schaltfläche zu klicken!$()
```